src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -6,10 +6,14 @@ import { Login } from "../Login/Login";
 
 export function PopupWithForm({
```

Comment:
please, make `PopupWithForm` the same way you had in the `React` project: it's name says: it is a popup with a form inside it. So, you need to have tag `form`, `submit button`, cross button inside `PopupWithForm`. And any content of this compment should be passed as its children.


It should be a universal compoent for any popups with a form. Not just for Register or Login